src/Core/src/Hosting/AppHost.cs

```diff
@@ -12,6 +12,7 @@ public static IAppHostBuilder CreateDefaultBuilder()
 
 			builder.UseMauiHandlers();
 			builder.ConfigureFonts();
+			builder.ConfigureImageSourceServices();
```

Comment:
Configuring the fonts is also using a Service, so, could be `ConfigureFontsService.` Could we use directly `ConfigureImageSources`?

Reply:
Done.